docs/Tutorials/Migrating-to-Tessera.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Migrating to Tessera
+description: Migrate your Orion configuration and data to Tessera.
+---
+
+# Migrating to Tessera
+
+## Migration Process
+
+A utility is included in Tessera which enables migration of an Orion config
+file and database to a Tessera config file and database. No changes are required to the config of Besu itself to migrate.
+
+A full migration workflow would be as follows:
+
+1. Download or build the migration utility
+1. Shut down the Orion and Hyperledger Besu nodes
+1. Perform the config and database migration
+1. Start Tessera with the new config and database files
+1. Start Hyperledger Besu nodes
+
+## Build Migration Utility
+
+The utility can be built from the [Tessera repository](https://github.com/ConsenSys/tessera/)
+
+1. First clone the Tessera repostory:
+
+`git clone https://github.com/ConsenSys/tessera`
+
+1. Navigate to the project root directory:
+
+`cd tessera`
+
+1. Build the migration utility using gradle:
+
+`./gradlew`
```

Comment:
`./gradlew` alone is is not building if you don't tell it what task to run.
Also if you run `./gradlew clean build` as the equivalent of the previously documented command but with the wrapper, then all tessera will build and test and we only need to build the `orion-to-tessera` module.
Plus it's not going to install the dist, meaning you will end up with compressed archives (tgz and zip) in the `migration/orion-to-tessera/build/distribution` directory so you would need to add instructions to expand them.
To do so, at the root of the project, run `./gradlew clean migration:orion-to-tessera:installDist -x test`
- `./gradlew` uses the wrapper
- `clean` purges all previous build artefacts
-  `migration:orion-to-tessera:` is the module that you want to target and only it
- `installDist` is the build task that will assemble a binary and expand the folders in `migration/orion-to-tessera/build/install/migration/` with `bin` and `lib` subfolders directly ready to run `./bin/migrate`.
- `-x test` will exclude the tests as we only need to build the binaries, not run the whole test suite that takes ages...

Reply:
thank you for the example and detail